webapp/src/mutator.ts

```diff
@@ -384,9 +385,17 @@ class Mutator {
         await undoManager.perform(
             async () => {
                 await octoClient.deleteBoardMember(member)
+                const user = await octoClient.getUser(member.userId)
+                if (user) {
+                    store.dispatch(removeBoardUsers([user]))
```

Comment:
Probably implementing this as `removeBoardUserById` or something like that, it would be less "homogeneous", but it would bet one server request less.

Reply:
Makes sense. Done.